sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java

```diff
@@ -190,7 +190,7 @@ public ModelsRepositoryAPIImplBuilder addPolicy(HttpPipelinePolicy customPolicy)
      * @return an instance of ModelsRepositoryAPIImpl.
      */
     public ModelsRepositoryAPIImpl buildClient() {
-        if(apiVersion == null){
+        if (apiVersion == null) {
```

Comment:
@srnagar is this an issue with codegen?

Reply:
@alzimmermsft are you referring to the formatting?

Reply:
Yeah, it was being flagged by Checkstyle linting

Reply:
yeah, I'll get this fixed in autorest.